src/app/core/models/advance-request-delete-params.model.ts

```
@@ -9,6 +8,6 @@ export interface AdvanceRequestDeleteParams {
   componentProps: {
     header: string;
     body: string;
-    deleteMethod: () => Observable<AdvanceRequests>;
+    deleteMethod: () => Observable<{}>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)